### PR TITLE
[#4551] fix error on fetching history for removed chat

### LINKED ISSUE #4551: Missing value for property 'transport.topic' error appeared after removing a public chat
### Description

*Type*: Bug

*Summary*: "Missing value for property 'transport.topic'" error appered when I removed a public chat. There was "Fetching messages.." in progress. Also, before that, I cleared the chat. When I closed the error, the app worked fine. I could not reproduce it anymore.

#### Expected behavior
No such error is shown.

#### Actual behavior
The error appeared. The app worked fine after it was dismissed. I could not reproduce it.

### Reproduction

There are no steps to reproduce. 
 
### Additional Information
* Status version: `release/0.9.19` [25](https://jenkins.status.im/job/status-react/job/release/job/release%252F0.9.19/25/), `08e0ee9`, May 31
* Operating System: happened on iOS

### PR DESCRIPTION
**Pull request by rasom**
fixes #4551 

status: ready

### CODE REVIEW

**Review by janherich — APPROVED** [review]

**Comment by rasom** [comment]
apk uploaded to https://i.diawi.com/PaKZ82 for branch PR-4555

<blockquote><div><strong><a href="https://i.diawi.com/PaKZ82">Install - Diawi - Development and In-house Apps Wireless Installation</a></strong></div></blockquote>

**Comment by rasom** [comment]
ipa uploaded to https://i.diawi.com/ak78zP for branch PR-4555

<blockquote><div><strong><a href="https://i.diawi.com/ak78zP">Install - Diawi - Development and In-house Apps Wireless Installation</a></strong></div></blockquote>

**Review by goranjovic — APPROVED** [review]

**Comment by rasom** [comment]
apk uploaded to https://i.diawi.com/KjHECR for branch PR-4555

<blockquote><div><strong><a href="https://i.diawi.com/KjHECR">Install - Diawi - Development and In-house Apps Wireless Installation</a></strong></div></blockquote>

**Comment by rasom** [comment]
ipa uploaded to https://i.diawi.com/LPLx1n for branch PR-4555

<blockquote><div><strong><a href="https://i.diawi.com/LPLx1n">Install - Diawi - Development and In-house Apps Wireless Installation</a></strong></div></blockquote>

File: src/status_im/transport/inbox.cljs [hunk 1]
```diff
@@ -357,7 +357,10 @@
 (handlers/register-handler-fx
  :inbox/check-fetching
  (fn [{:keys [db now] :as cofx} [_ last-request chat-id]]
-   (when (:inbox/fetching? db)
+   (when (and (:inbox/fetching? db)
+              ;; if chat was removed before messages were fetched no need
+              ;; to proceed with further actions
+              (or (not chat-id) (contains? (:transport/chats db) chat-id)))
```
Comment (yenda) [inline src/status_im/transport/inbox.cljs]: this event can also be called for fetching of all chats, when chat-id is `nil`, (contains? m nil) is false

**Review by yenda — APPROVED** [review]

**Comment by rasom** [comment]
apk uploaded to https://i.diawi.com/dpgFBe for branch PR-4555

**Comment by rasom** [comment]
ipa uploaded to https://i.diawi.com/o8ry39 for branch PR-4555

**Review by yenda — APPROVED** [review]

**Pull request merged**